### PR TITLE
update csi-snapshotter image tag

### PR DESCRIPTION
**Pull request by raunakkumar**
Problem: According to this table: https://kubernetes-csi.github.io/docs/external-snapshotter.html 4.1.x is not deemed stable
Implementation: Switch to the stable image tags from the above table

Signed-off-by: Raunak <raunak.kumar@hpe.com>

### CODE REVIEW

File: docs/index.yaml [hunk 1]
```diff
@@ -331,7 +331,8 @@ entries:
   - apiVersion: v1
     appVersion: "3.1"
     created: "2021-05-06T12:00:48.369807-07:00"
-    description: A Helm chart for installing the HPE Volume Driver for Kubernetes FlexVolume plugin
+    description: A Helm chart for installing the HPE Volume Driver for Kubernetes
```
Comment (sneharai4) [inline docs/index.yaml:334]: Why do we see changes here?

File: docs/index.yaml [hunk 2]
```diff
@@ -354,7 +355,8 @@ entries:
   - apiVersion: v1
     appVersion: "3.0"
     created: "2020-05-13T15:47:43.269014-04:00"
-    description: A Helm chart for installing the HPE Volume Driver for Kubernetes FlexVolume plugin
+    description: A Helm chart for installing the HPE Volume Driver for Kubernetes
```
Comment (sneharai4) [inline docs/index.yaml:358]: Is this change needed?

File: docs/index.yaml (hunk 1 above)
Comment (raunakkumar) [reply docs/index.yaml:334]: Its probably we have different helm version and it wraps the description for some of us which udpates timestamp. We have seen in prior build as well. Its involuntary 
```
 helm version
version.BuildInfo{Version:"v3.2.4", GitCommit:"0ad800ef43d3b826f31a5ad8dfbb4fe05d143688", GitTreeState:"dirty", GoVersion:"go1.14.3"
```

**Review by sneharai4 — APPROVED** [review]
